File: questions/Python/Basic/Es1.py
# -- "Easy", weight: '5'
# 'Realizza una funzione "run" '
# 'che sommi due numeri.'

import unittest
from RunningFile import run


class Test(unittest.TestCase):
    def test_1(self):
        self.assertEqual(run(1, 2), 3, 'Errore')

    def test_2(self):
        self.assertEqual(run(0, 2), 2, 'Errore')

    def test_3(self):
        self.assertEqual(run(-1, 2), 1, 'Errore')


unittest.main()

File: questions/Java/Basic/Es1.java
/*Easy-'Realizza una classe Esercizio a cui si passi nel costruttore due numeri interi, tale classe deve avere una funzione
run che ritorni la somma dei due numeri'*/


public class Es1{
    public static void main(String[] args) {
	    Sum es = new Sum(1,2);
        if (es.run() == 3){
            System.out.println("Test eseguiti con successo.");
        }
        else {
            System.out.println("Test falliti.");
        }
    }
}